SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -6,7 +6,8 @@
 #include "Engine/EngineTypes.h"
 #include "Misc/Paths.h"
 #include "Utils/SpatialActorGroupManager.h"
-#include "Utils/SpatialDebugger.h"
+#include "LoadBalancing/AbstractLBStrategy.h"
```

Comment:
Any reason you're including these here and not forward declaring?

Reply:
I don't know if TSubclassOf requires a complete type or not.
It might have only worked before because there was some other include in cpp files that used this that made it work. If it's not needed we can revert it though.